test/spec/vis/vis.spec.js

```diff
@@ -444,43 +446,73 @@ describe("Vis", function() {
 
   describe("dragging option", function() {
 
-    it("should be enabled with zoom overlay and scrollwheel enabled", function() {
+    beforeEach(function() {
       var container = $('<div>').css('height', '200px');
-      var vis = new cdb.vis.Vis({el: container});
+      this.vis = new cdb.vis.Vis({el: container});
+    });
 
-      var mapConfig = {
-        updated_at: 'cachebuster',
-        title: "irrelevant",
-        url: "http://cartodb.com",
-        center: [40.044, -101.95],
-        bounding_box_sw: [20, -140],
-        bounding_box_ne: [ 55, -50],
-        zoom: 4,
-        bounds: [[1, 2],[3, 4]],
+    it("should be enabled with zoom overlay and scrollwheel enabled", function() {
+      var mapConfig = createMapConfig({
         scrollwheel: true,
-        overlays: [
-          {
-            type: "zoom",
-            order: 6,
-            options: {
-              x: 20,
-              y: 20,
-              display: true
-            },
-            template: ""
-          }
-        ],
-      };
-
-      vis.load(mapConfig);
-      expect(vis.map.get('drag')).toBeTruthy();
+        zoom: true
+      });
+      this.vis.load(mapConfig);
+      expect(this.vis.map.get('drag')).toBeTruthy();
     });
 
     it("should be enabled with zoom overlay and scrollwheel disabled", function() {
-      var container = $('<div>').css('height', '200px');
-      var vis = new cdb.vis.Vis({el: container});
+      var mapConfig = createMapConfig({
+        scrollwheel: false,
+        zoom: true
+      });
+      this.vis.load(mapConfig);
+      expect(this.vis.map.get('drag')).toBeTruthy();
+    });
+
+    it("should be enabled without zoom overlay and scrollwheel enabled", function() {
+      var mapConfig = createMapConfig({
+        scrollwheel: true,
+        zoom: false
+      });
+      this.vis.load(mapConfig);
+      expect(this.vis.map.get('drag')).toBeTruthy();
+    });
+
+    it("should be disabled without zoom overlay and scrollwheel disabled", function() {
+      var mapConfig = createMapConfig({
+        scrollwheel: false,
+        zoom: false
+      });
+      this.vis.load(mapConfig);
+      expect(this.vis.map.get('drag')).toBeFalsy();
+    });
+
+    describe('over mobile', function() {
+      beforeEach(function() {
+        spyOn(this.vis, 'isMobileDevice').and.returnValue(true);
+      });
+
+      it("should be enabled if scrollwheel is enabled (no matter zoom overlay) but device is mobile", function() {
```

Comment:
We're missing some tests for other possible combinations here. Example: `{ scrollwheel: false, zoom: true }` and `{ scrollwheel: true, zoom: true }`. I think we're missing one when isMobileDevice is false.